packages/sources/accuweather/test/integration/location-current-conditions.ts

```diff
@@ -0,0 +1,180 @@
+        const response = await req
+          .post('/')
+          .send(data)
+          .set('Accept', '*/*')
+          .set('Content-Type', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+
+        assertSuccess({ expected: 200, actual: response.statusCode }, response.body, id)
+        expect(response.body).toMatchSnapshot()
+      })
+
+      it('returns the result as JSON', async () => {
+        const data: AdapterRequest = {
+          id,
+          data: {
+            endpoint: 'location-current-conditions',
+            lat: 0,
+            lon: 0,
+            units: Unit.METRIC,
+            encodeResult: false,
+          },
+        }
+        mockAWLocationResponseSuccessLocationNotFound()
+
+        const response = await req
+          .post('/')
+          .send(data)
+          .set('Accept', '*/*')
+          .set('Content-Type', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+
+        assertSuccess({ expected: 200, actual: response.statusCode }, response.body, id)
+        expect(response.body).toMatchSnapshot()
+      })
+    })
+
+    describe('when a location is found for the given coordinates', () => {
+      it('returns the result encoded', async () => {
+        const data: AdapterRequest = {
+          id,
+          data: {
+            endpoint: 'location-current-conditions',
+            lat: '40.78136100040876',
+            lon: '-77.89687509335249',
+            units: Unit.METRIC,
+          },
+        }
+        mockAWLocationResponseSuccessLocationFound()
+        mockAWCurrentConditionsResponseSuccess()
+
+        const response = await req
+          .post('/')
+          .send(data)
+          .set('Accept', '*/*')
+          .set('Content-Type', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+
+        assertSuccess({ expected: 200, actual: response.statusCode }, response.body, id)
+        expect(response.body).toMatchSnapshot()
+      })
+
+      it('returns the result as JSON', async () => {
+        const data: AdapterRequest = {
+          id,
+          data: {
+            endpoint: 'location-current-conditions',
+            lat: '40.78136100040876',
+            lon: '-77.89687509335249',
+            units: Unit.METRIC,
+            encodeResult: false,
+          },
+        }
+        mockAWLocationResponseSuccessLocationFound()
+        mockAWCurrentConditionsResponseSuccess()
+
+        const response = await req
+          .post('/')
+          .send(data)
+          .set('Accept', '*/*')
+          .set('Content-Type', 'application/json')
+          .expect('Content-Type', /json/)
+          .expect(200)
+
+        assertSuccess({ expected: 200, actual: response.statusCode }, response.body, id)
+        expect(response.body).toMatchSnapshot()
+      })
+    })
+  })
+}
+
+// no location found as JSON
+// no location found encoded
+// found as JSON
+// found encoded
```

Comment:
Are these suggestions for future tests? Maybe you can move these to the top and make it explicit?

Reply:
Oh, just comments to be removed.